# Welcome to Cantini Lab!


Single-cell high-throughput sequencing, a major breakthrough in life sciences, allows us to access the integrated molecular profiles of thousands of cells in a single experiment. This abundance of data provides tremendous power to unveil unknown cellular mechanisms. However, single-cell data are so massive and complex that it has become challenging to give clues to their underlying biological processes.

The [machine learning for integrative genomics G5 group](https://research.pasteur.fr/en/team/machine-learning-for-integrative-genomics/) works at the interface of machine learning and genomics, developing methods exploiting the full richness and complementarity of the available single-cell data to derive actionable biological knowledge.


# Resources

***
| [<img alt="Mowgli" src="https://github.com/cantinilab/.github/blob/main/profile/logos/mowgli.png" width="600"/>](https://github.com/cantinilab/Mowgli) | [<img alt="scConfluence" src="https://github.com/cantinilab/.github/blob/main/profile/logos/scconfluence.png" width="380"/>](https://github.com/cantinilab/scconfluence) | [<img alt="HuMMuS" src="https://github.com/cantinilab/.github/blob/main/profile/logos/hummus.png" height="20"/>](https://github.com/cantinilab/HuMMuS) | [<img alt="scPrint" src="https://github.com/cantinilab/.github/blob/main/profile/logos/scprint.png" width="500"/>](https://github.com/cantinilab/scPrint) |
| --- | --- | --- | --- |
| **MOWGLI**: [Integrating paired multimodal single-cell data](https://doi.org/10.1038/s41467-023-43019-2) | **scConfluence**: [Integrating unpaired multimodal single-cell data](https://doi.org/10.1038/s41467-024-51382-x) | **HuMMuS**: [Molecular mechanisms from multi-omics single-cell data](https://doi.org/10.1093/bioinformatics/btae143) | **scPrint**: [Transcriptomic foundation model for gene network inference and more](https://doi.org/10.1101/2024.07.29.605556) |
| [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/home.svg" height="16">](https://mowgli.readthedocs.io/en/latest/) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/python.svg" height="16">](https://github.com/cantinilab/Mowgli) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/package.svg" height="16"><sub><sup> PYPI</sup></sub>](https://pypi.org/project/mowgli/) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/article.svg" height="16">](https://doi.org/10.1093/bioinformatics/btae143) | [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/home.svg" height="16">](https://scconfluence.readthedocs.io/en/latest/) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/python.svg" height="16">](https://github.com/cantinilab/scConfluence) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/package.svg" height="16"><sub><sup> PYPI</sup></sub>](https://pypi.org/project/scconfluence/) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/article.svg" height="16">](https://doi.org/10.1038/s41467-023-43019-2) | [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/home.svg" height="16">](https://cantinilab.github.io/HuMMuS/) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/r.svg" height="16">](https://github.com/cantinilab/HuMMuS) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/package.svg" height="16"><sub><sup> PYPI</sup></sub>](https://pypi.org/project/hummuspy/) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/article.svg" height="16">](https://doi.org/10.1038/s41467-024-51382-x) | [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/home.svg" height="16">](https://www.jkobject.com/scPRINT/) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/python.svg" height="16">](https://github.com/cantinilab/scPrint) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/package.svg" height="16"><sub><sup> PYPI</sup></sub>](https://pypi.org/project/scprint/) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/article.svg" height="16">](https://doi.org/10.1101/2024.07.29.605556) |
[<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/stories.png" width="600">](https://github.com/cantinilab/stories) | [<img src="https://raw.githubusercontent.com/cantinilab/circe/main/logo_dark_theme.svg" width="600">](https://github.com/cantinilab/circe) | [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/momix.jpg" width="600">](https://github.com/cantinilab/momix-notebook) | 
**stories**: [Learning cell fate landscapes from spatial transcriptomics using Fused Gromov-Wasserstein](https://doi.org/10.1101/2024.07.26.605241) | **CIRCE**: Predict cis-regulatory interactions between DNA regions  | **MOMIX**: [Benchmark of multi-omics joint Dimensionality Reduction (jDR) approaches in cancer study](https://doi.org/10.1038/s41467-020-20430-7) |
| [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/home.svg" height="16">](https://stories.readthedocs.io/en/latest/) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/python.svg" height="16">](https://github.com/cantinilab/stories) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/package.svg" height="16"><sub><sup> PYPI</sup></sub>](https://pypi.org/project/stories-jax/) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/article.svg" height="16">](https://doi.org/10.1101/2024.07.26.605241) | [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/python.svg" height="14">](https://github.com/cantinilab/circe) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/package.svg" height="16"><sub><sup> PYPI</sup></sub>](https://pypi.org/project/circe-py/) | [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/r.svg" height="16">](https://github.com/cantinilab/momix-notebook) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/article.svg" height="16">](https://doi.org/10.1038/s41467-020-20430-7)
  


### Other resources
***
| [<img alt="scDataLoader" src="https://github.com/cantinilab/.github/blob/main/profile/logos/scdataloader.png" width="250"/>](https://github.com/jkobject/scdataloader) | [<img alt="benGRN" src="https://github.com/cantinilab/.github/blob/main/profile/logos/bengrn.png" width="100"/>](https://github.com/jkobject/bengrn) | [<img alt="GRnnData" src="https://github.com/cantinilab/.github/blob/main/profile/logos/grnndata.png" width="100"/>](https://github.com/cantinilab/grnndata) |
| --- | --- | --- |
| **scDataLoader**: a dataloader to work with large single cell datasets from lamindb | **benGRN**: Awesome Benchmark of Gene Regulatory Networks | **GRnnData**: Awesome GRN enhanced AnnData toolkit |
| [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/python.svg" height="16">](https://github.com/jkobject/scdataloader) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/package.svg" height="16"><sub><sup> PYPI</sup></sub>](https://pypi.org/project/scdataloader/) | [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/python.svg" height="16">](https://github.com/jkobject/bengrn) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/package.svg" height="16"><sub><sup> PYPI</sup></sub>](https://pypi.org/project/bengrn/) | [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/python.svg" height="16">](https://github.com/cantinilab/grnndata) [<img src="https://github.com/cantinilab/.github/blob/main/profile/logos/package.svg" height="16"><sub><sup> PYPI</sup></sub>](https://pypi.org/project/grnndata/) |
